weaviate/collections/classes/filters.py

```diff
@@ -182,3 +185,305 @@ def like(self, val: str) -> _FilterValue:
+        """Filter on whether the last update time is less than the given time.
+
+        Arguments:
+            `date`
+                date to filter on.
+            `on_reference_path`
+                If the filter is on a cross-ref property, the path to the property to be filtered on, example: on_reference_path=["ref_property", "target_collection"].
+        """
+        return _FilterTime.less_than(date, _FilterUpdateTime._prepare_path(on_reference_path))
+
+    @staticmethod
+    def less_or_equal(date: DATE, on_reference_path: Optional[List[str]] = None) -> _FilterValue:
+        """Filter on whether the last update time is less than or equal to the given time.
+
+        Arguments:
+            `date`
+                date to filter on.
+            `on_reference_path`
+                If the filter is on a cross-ref property, the path to the property to be filtered on, example: on_reference_path=["ref_property", "target_collection"].
+        """
+        return _FilterTime.less_or_equal(date, _FilterUpdateTime._prepare_path(on_reference_path))
+
+    @staticmethod
+    def greater_than(date: DATE, on_reference_path: Optional[List[str]] = None) -> _FilterValue:
+        """Filter on whether the last update time is greater than the given time.
+
+        Arguments:
+            `date`
+                date to filter on.
+            `on_reference_path`
+                If the filter is on a cross-ref property, the path to the property to be filtered on, example: on_reference_path=["ref_property", "target_collection"].
+        """
+        return _FilterTime.greater_than(date, _FilterUpdateTime._prepare_path(on_reference_path))
+
+    @staticmethod
+    def greater_or_equal(date: DATE, on_reference_path: Optional[List[str]] = None) -> _FilterValue:
+        """Filter on whether the last update time is greater than or equal to the given time.
+
+        Arguments:
+            `date`
+                date to filter on.
+            `on_reference_path`
+                If the filter is on a cross-ref property, the path to the property to be filtered on, example: on_reference_path=["ref_property", "target_collection"].
+        """
+        return _FilterTime.greater_or_equal(
+            date, _FilterUpdateTime._prepare_path(on_reference_path)
+        )
+
+    @staticmethod
+    def _prepare_path(path: Optional[List[str]]) -> List[str]:
+        return path or [] + ["_lastUpdateTimeUnix"]
+
+
+class FilterMetadata:
+    """Define a filter based on a ID, Creation- or LastUpdateTime  to be used when querying a collection.
+
+    To combine multiple filters, you can use `&` or `|` operators for each `AND` or `OR` operations, e.g.,
+        `FilterMetadata.FilterById.equal(UUID) & Filter("age").greater_than(18)`
+
+    See [the docs](https://weaviate.io/developers/weaviate/search/filters) for more details!
+    """
+
+    FilterById = _FilterId
+    FilterByCreationTime = _FilterCreationTime
+    FilterByUpdateTime = _FilterUpdateTime
```

Comment:
`FilterMetadata.FilterById` seems a bit long-winded compared to the properties filter: `Filter("age").greater_than(18)`. WDYT about having `FilterMetadata.ById` or `FilterMetadata.by_id()` instead?